frontend/components/droplets/droplet-tile.tsx

```
@@ -12,7 +12,10 @@ import { toast } from "sonner";
 import { Archive, ArchiveRestore, Clock } from "lucide-react";
 import { getDueDateBadgeColor } from "@/lib/utils";
 import { DateTime } from "luxon";
-import { archiveDroplet } from "@/lib/requests/droplet";
+import { archiveDroplet, favoriteDroplet } from "@/lib/requests/droplet";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import { grey, purple } from "@mui/material/colors";
```

ESLint

[error] 'grey' is defined but never used.

(@typescript-eslint/no-unused-vars)